pkg/kubelet/secret/secret_manager.go

```diff
@@ -111,6 +112,19 @@ const (
 	defaultTTL = time.Minute
 )
 
+func getTPClient(kubeClients []clientset.Interface, tenant string) clientset.Interface {
```

Comment:
I thought we moved this function to a common util function, no ?

Reply:
will do later as described in the description. this change is fixing a blocking issue so let's have it in first.